e2e-tests/tests/governed_map/smart_contracts/test_get_insert_remove.py

```diff
@@ -0,0 +1,94 @@
+from pytest import fixture, mark
+from src.blockchain_api import BlockchainApi
+from conftest import string_to_hex_bytes, hex_bytes_to_string
+
+
+class TestGet:
+    def test_insert_returncode(self, insert_data):
+        assert 0 == insert_data.returncode
+
+    @mark.usefixtures("insert_data")
+    def test_get_returncode(self, api: BlockchainApi, random_key):
```

Comment:
I would like tests to not grow the map without bounds. We know that this functionality was intended to keep tens of values.

Reply:
thanks, fixed in `insert_data` fixture